administrator-manual/en/base_system.rst

```diff
@@ -248,10 +248,13 @@ configuring the network clients to use it. As alternatives, the
 * requesting a new *Let's Encrypt* [#Letsencrypt]_ certificate.  This is
   possible if the following requirements are met:
 
-  1. the server must be reachable from outside at port 80. Make sure your port 80
+  1. The server must be reachable from outside at port 80. Make sure your port 80
      is open to the public Internet (you can check with sites like [#CSM]_);
+     
+# I could be wrong, doesn't letsencrypt need access to port 443 to generate the certificate and renew the certificate?
```

Comment:
Let's encrypt only needs port 80.

Remove this comments and we will merge the PR.
Really good job!